source/users/create.txt

```diff
@@ -1,41 +1,108 @@
-=============
-Create a User
-=============
+==================
+Create an App User
+==================
 
 .. default-domain:: mongodb
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-Create a User Upon First Authentication
----------------------------------------
+Overview
+--------
 
-For most :ref:`authentication providers <authentication-providers>`, 
-Atlas App Services automatically creates a :ref:`user object <user-objects>` the 
-first time a user authenticates through the authentication provider. The 
-only exception is :ref:`email/password user authentication 
-<email-password-authentication>`, in which you must register and confirm 
-a user before the user can authenticate.
+Atlas App Services provides various :ref:`authentication providers <authentication-providers>`
+to log users into your app. For most providers, App Services automatically creates a 
+user account the first time a user authenticates through the provider. The only exception is 
+:ref:`email/password user authentication <create-email-password-user>`, which requires 
+registering and confirming a user before the user can authenticate.
 
 .. include:: /includes/apple-account-deletion-requirements.rst
 
+Identity
+--------
+
+Upon first login with an authentication provider, App Services creates a 
```

Comment:
Let's be more direct here. We may want to consider, 'who is first logging in with an authentication provider'. I think a better way to format this sentence would be: 

"When **you** first log in with an authentication provider, ..."

Not required, but good supplemental reading about 2nd person usage (if you're interested): https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/